[msx] test Started org.apache.hadoop.hdfs.server.datanode.TestDataNodeVolumeFailureReporting#testMultipleVolFailuresOnNode
2019-12-31 19:49:17,880 [Thread-408] INFO  namenode.NameNode (NameNode.java:performReconfig(961)) - [msx-restart] NameNode init, reconfigMode is none, do nothing
2019-12-31 19:49:18,078 [Thread-408] INFO  namenode.NameNode (NameNode.java:<init>(1039)) - [msx-restart] NameNode start
2019-12-31 19:49:18,090 [Thread-408] INFO  datanode.DataNode (DataNode.java:performReconfig(463)) - [msx-restart] DataNode init, reconfigMode is none, do nothing
2019-12-31 19:49:18,164 [Thread-408] INFO  datanode.DataNode (DataNode.java:runDatanodeDaemon(2732)) - [msx-restart] DataNode start
2019-12-31 19:49:18,513 [Thread-408] INFO  datanode.DataNode (DataNode.java:shutdown(2072)) - [msx-restart] DataNode stop, double check before stop, dfs.block.access.token.enable = false
2019-12-31 19:49:18,723 [Thread-408] INFO  namenode.NameNode (NameNode.java:stop(1082)) - [msx-restart] NameNode stop, double check before stop, dfs.block.access.token.enable = false
2019-12-31 19:49:18,895 [Thread-408] INFO  namenode.NameNode (NameNode.java:performReconfig(961)) - [msx-restart] NameNode init, reconfigMode is none, do nothing
2019-12-31 19:49:19,051 [Thread-408] INFO  namenode.NameNode (NameNode.java:<init>(1039)) - [msx-restart] NameNode start
2019-12-31 19:49:19,061 [Thread-408] INFO  datanode.DataNode (DataNode.java:performReconfig(463)) - [msx-restart] DataNode init, reconfigMode is none, do nothing
2019-12-31 19:49:19,106 [Thread-408] INFO  datanode.DataNode (DataNode.java:runDatanodeDaemon(2732)) - [msx-restart] DataNode start
2019-12-31 19:49:19,134 [Thread-408] INFO  datanode.DataNode (DataNode.java:performReconfig(463)) - [msx-restart] DataNode init, reconfigMode is none, do nothing
2019-12-31 19:49:19,184 [Thread-408] INFO  datanode.DataNode (DataNode.java:runDatanodeDaemon(2732)) - [msx-restart] DataNode start
2019-12-31 19:49:19,196 [Thread-408] INFO  datanode.DataNode (DataNode.java:performReconfig(463)) - [msx-restart] DataNode init, reconfigMode is none, do nothing
2019-12-31 19:49:19,245 [Thread-408] INFO  datanode.DataNode (DataNode.java:runDatanodeDaemon(2732)) - [msx-restart] DataNode start
2019-12-31 19:49:31,774 [Thread-408] INFO  datanode.DataNode (DataNode.java:shutdown(2072)) - [msx-restart] DataNode stop, double check before stop, dfs.block.access.token.enable = false
2019-12-31 19:49:31,835 [Thread-408] INFO  datanode.DataNode (DataNode.java:shutdown(2072)) - [msx-restart] DataNode stop, double check before stop, dfs.block.access.token.enable = false
2019-12-31 19:49:31,878 [Thread-408] INFO  datanode.DataNode (DataNode.java:shutdown(2072)) - [msx-restart] DataNode stop, double check before stop, dfs.block.access.token.enable = false
2019-12-31 19:49:32,045 [Thread-408] INFO  namenode.NameNode (NameNode.java:stop(1082)) - [msx-restart] NameNode stop, double check before stop, dfs.block.access.token.enable = false
[msx] test Finished org.apache.hadoop.hdfs.server.datanode.TestDataNodeVolumeFailureReporting#testMultipleVolFailuresOnNode test result = 1
